interface/reports/collections_report.php

```diff
@@ -1033,7 +1054,7 @@ function checkAll(checked) {
 <?php
       if ($form_age_cols) {
         for ($c = 0; $c < $form_age_cols; ++$c) {
-          echo "  <td class='detail' align='right'>";
+          echo "<td class='detail <?php echo $form_cb_with_debt && $balance<=0?\"delete\":\"\";?>' align='right'>";
```

Comment:
lets place some spacing and brackets so mere mortals can read this :)
maybe something like:
``````
echo "<td class='detail <?php echo ($form_cb_with_debt && $balance<=0) ? \"delete\" : \"\"; ?>' align='right'>";